### PR TITLE
databrowser windowdressing

### PR DESCRIPTION
**Pull request by schoffelen**
working on the tutorial for the cuttingeeg symposium I came across a few esthetic issues in databrowser which may be easy to address

### CODE REVIEW

File: ft_databrowser.m [hunk 1]
```diff
@@ -49,6 +49,8 @@
 %   cfg.visible                 = string, 'on' or 'off' whether figure will be visible (default = 'on')
 %   cfg.position                = location and size of the figure, specified as a vector of the form [left bottom width height]
 %   cfg.renderer                = string, 'opengl', 'zbuffer', 'painters', see MATLAB Figure Properties. If this function crashes, you should try 'painters'.
+%   cfg.colormap                = string, or Nx3 matrix, colormap for the
```
Comment (robertoostenveld) [inline ft_databrowser.m:52]: can you put the help on a single line (it is not that long) and can you make it consistent with other ft functions?

```
mac036-lan> cd fieldtrip
mac036-lan> grep "%   cfg.colormap" *.m
ft_multiplotTFR.m:%   cfg.colormap         = any sized colormap, see COLORMAP
ft_singleplotTFR.m:%   cfg.colormap       = any sized colormap, see COLORMAP
ft_topoplotER.m:%   cfg.colormap           = any sized colormap, see COLORMAP
ft_topoplotIC.m:%   cfg.colormap           = any sized colormap, see COLORMAP
ft_topoplotTFR.m:%   cfg.colormap           = any sized colormap, see COLORMAP
```

File: ft_databrowser.m [hunk 2]
```diff
@@ -658,6 +663,16 @@
 % open a new figure with the specified settings
 h = open_figure(keepfields(cfg, {'figure', 'position', 'visible', 'renderer'}));
 
+% check colormap is in proper format and set it
+if isfield(cfg, 'colormap')
+  if ischar(cfg.colormap)
+    cfg.colormap = ft_colormap(cfg.colormap);
+  elseif iscell(cfg.colormap)
+    cfg.colormap = ft_colormap(cfg.colormap{:});
```
Comment (robertoostenveld) [inline ft_databrowser.m:671]: so this allows to pass the number of steps, right? I see that this is also how it is done in topoplot_common. Can you check that all uses of ft_colormap work consistently? That also means that the check

```
  if size(cfg.colormap,2)~=3
    ft_error('cfg.colormap must be Nx3');
  end
```

from topoplot_common should be added here.

**Review by robertoostenveld — APPROVED** [review]

File: ft_databrowser.m (hunk 2 above)
Comment (schoffelen) [reply ft_databrowser.m:670]: OK, done. The ft_topo<something> rely on topoplot_common, so no action needed there. I addressed it in the relevant multi<> and single<>

**Comment by schoffelen** [comment]
I think that this one in principle is ready to be merged. Before doing so, I will quickly do a sweep of the open bugs on bugzilla that are linked to the mother of all databrowser bug. If there's stuff that can be relatively easily addressed, I'll consider it. Other issues I will set to 'wontfix' due to limited interest in the past 10 years or so....

**Comment by schoffelen** [comment]
http://bugzilla.fieldtriptoolbox.org/show_bug.cgi?id=2314

**Pull request merged**